server/gitrest/packages/gitrest-base/src/utils/gitWholeSummaryManager.ts

```diff
@@ -322,7 +515,10 @@ export class GitWholeSummaryManager {
         return blobResponse.sha;
     }
 
-    private async getShaFromTreeHandleEntry(entry: IWholeSummaryTreeHandleEntry): Promise<string> {
+    private async getShaFromTreeHandleEntry(
```

Comment:
Should we use the argument repoManager as opposed to this.repoManager?

Reply:
I should remove3 the repoManager param here. Using `this.repoManager` in this method is intentional, because referenced summaries exist in storage, not in memory, so we never want to accidentally use the in-memory repo for this particular method.

Reply:
Removed unused param and added explanatory comments